gui/resources.xml

```diff
@@ -490,14 +503,15 @@ Vocabulary
           <property name="tooltip" translatable="yes" context="Accel Editor (descriptions)">Remove the current layer from the layer stack.</property>
           <signal name="activate" handler="remove_layer_cb"/>
         </object>
-        <accelerator key="Delete" modifiers="GDK_SHIFT_MASK"/>
+        <accelerator key="Delete" modifiers="GDK_CONTROL_MASK"/>
       </child>
       <child>
         <object class="GtkAction" id="NormalizeLayerMode">
           <property name="label" translatable="yes" context="Menu→Layer (labels), Accel Editor (labels)">Convert to Normal Painting Layer</property>
           <property name="tooltip" translatable="yes" context="Accel Editor (descriptions)">Convert the current layer or group to a painting layer in ‘Normal’ mode, preserving its appearance.</property>
           <signal name="activate" handler="normalize_layer_mode_cb"/>
         </object>
+        <accelerator key="n" modifiers="GDK_CONTROL_MASK | GDK_SHIFT_MASK"/>
```

Comment:
I think this needs a use case for a keyboard shortcut, even a heavily modified one. I'd be happy leaving it as an advanced feature which can only be engaged by visiting the Layers panel or the menus.

(Normalize is weird and finicky, because it sometimes picks up background elements if those are made to look different when you normalize a layer. Users can be surprised by this because it may not be immediately apparent. Let's make advanced things that have to be thought about specially _look_ rare and advanced by not assigning them keys (although all that meta-alt-ctrl-shift-splarg-doubledeclutch-N stuff serves much the same purpose tbh!))